CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ For information on changes in released versions of Teku, see the [releases page]
 - Support v.2.1.0 of the standard rest api. it should be noted that the 'version' has been changed to lower case to comply with the api specification.
 - Attestations are now sent to the beacon node in batches by default when using the validator-client.
 - Updated to Javalin 4 for the rest api.
+- Aggregate attestation messages with the same attestation root are no longer discarded.
```

Comment:
This isn't particularly accurate - it only affects gossip handling but this suggests we were not processing attestations we should have.  I'd suggest:
```suggestion
- Removed ignore rule for aggregate attestation gossip where the attestation root has previously been seen
```
```